src/gui/mainwindow.cpp

```diff
@@ -1537,13 +1534,7 @@ void MainWindow::addUnauthenticatedTracker(const QPair<BitTorrent::TorrentHandle
 // Called when a tracker requires authentication
 void MainWindow::trackerAuthenticationRequired(BitTorrent::TorrentHandle *const torrent)
```

Comment:
This method is unused now. And I think the same goes for the one above it and the `m_unauthenticatedTrackers` variable.
Although it is weird that we don't support tracker logins anymore.

Reply:
qBittorrent supported plain text authentication only (using libtorrent). libtorrent stopped to support it since v1.1 since it is deprecated.

Reply:
>This method is unused now. And I think the same goes for the one above it and the m_unauthenticatedTrackers variable.

Removed.